nengo_loihi/simulator.py

```diff
@@ -142,7 +142,7 @@ class Simulator(object):
     unsupported = []
 
     def __init__(self, network, dt=0.001, seed=None, model=None,  # noqa: C901
-                 precompute=False, target=None):
+                 precompute=False, target=None, loihi_args=None):
```

Comment:
I really would like to limit the amount of args you can pass to the simulator. Everyone using Nengo Loihi is going to interact with the `Simulator`, so its interface needs to be as polished as possible. I feel like only devs are going to really want to futz with args passed to `LoihiSimulator`, and I believe the max spikes per step thing is removed in some PR in the queue, so figuring out an external interface to it isn't necessary.

Could we drop this commit from the PR? If you feel strongly about keeping it, then maybe have it be a separate PR?

Reply:
I need this for getting all larger convnets working, since they typically have high input spike bandwidth so I need to adjust `snip_max_spikes_per_step`. How about we add it for now, and then if/when we do remove `snip_max_spikes_per_step` then we can get rid of this? I can make it another PR if you'd prefer, but I don't think that's going to speed things along too much, because we'd still be waiting on that for the convnet stuff. I guess it would let us move on to the communication refactoring and learning.